index.html

```diff
@@ -1,11 +1,129 @@
+      </h1>
+
+      <button type="button" class="hero-button">
+        Замовити послугу
+      </button>
+    </section>
+
+    <section class="features">
+      <div class="container">
+        <ul class="features-list">
+          <li class="features-item feature">
+            <h3 class="feature-title">Увага до деталей</h3>
+            <p class="feature-text">Ідейні міркування, і навіть початок повсякденної роботи з формування позиції.</p>
+          </li>
+          <li class="features-item feature">
+            <h3 class="feature-title">Пунктуальність</h3>
+            <p class="feature-text"> Завдання організації, особливо рамки і місце навчання кадрів тягне у себе.</p>
+          </li>
+          <li class="features-item feature">
+            <h3 class="feature-title">Планування</h3>
+            <p class="feature-text">Так само консультація з широким активом значною мірою зумовлює.</p>
+          </li>
+          <li class="features-item feature">
+            <h3 class="feature-title">Сучасні технології</h3>
+            <p class="feature-text">Значимість цих проблем настільки очевидна, що реалізація планових завдань.</p>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="services">
+      <!-- container-->
+      <h2>Чим ми займаємось</h2>
+
+      <ul class="activities">
+        <li>
+          <img class="activities-item" src="./images/plan.jpg" width="370" height="294" alt="plan">
+        </li>
+        <li>
+          <img class="activities-item" src="./images/phone.jpg" width="370" height="294" alt="size">
+        </li>
+        <li>
+          <img class="activities-item" src="./images/colors.jpg" width="370" height="294" alt="design">
+        </li>
+      <!-- container-->
+
+    </section>
+
+    <section class="team">
+      <h2 class="team-title">Наша команда</h2>
+
+      <ul class="team-list">
+        <li class="team-item person">
+          <img class="person-photo" src="./images/demyanenko.jpg" width="270" height="260" alt="Igor Demyanenko">
+          <h3 class="person-name">Igor Demyanenko </h3>
+          <p class="person-position">Product designer</p>
+        </li>
+        <li>
+          <img class="person-photo" src="./images/repina.jpg" width="270" height="260" alt="Olga Repina">
+          <h3 class="person-name">Olga Repina</h3>
+          <p class="person-position">Frontend develover</p>
+        </li>
+        <li>
+          <img class="person-photo" src="./images/tarasov.jpg" width="270" height="260" alt="Mykola Tarasov">
+          <h3 class="person-name">Mykola Tarasov</h3>
+          <p class="person-position">Marketing</p>
+        </li>
+        <li>
+          <img class="person-photo" src="./images/ermakov.jpg" width="270" height="260" alt="Myhailo Ermakov">
+          <h3 class="person-name">Myhailo Ermakov</h3>
+          <p class="person-position">UI designer</p>
+        </li>
+      </ul>
+    </section>
+  </main>
+
+  <footer>
```

Comment:
add class to `footer` element

Reply:
Why need we add class to footer, if we can use .footer? Is it necessary?

Reply:
it is as we use classes as selectors across the project to avoid specificity issues